packages/transactions/package.json

```diff
@@ -32,8 +32,6 @@
     "long": "^4.0.0",
     "path": "^0.12.7"
   },
-  "devDependencies": {
-    "@helium/crypto": "^3.1.0"
```

Comment:
Should we add this as a peer dependency so that users know its required?

Reply:
It's required in `"dependencies"`
https://github.com/helium/helium-js/blob/afceb7fe6f07e178874a93af5ddc3e4b9ebea862/packages/transactions/package.json#L30

I didn't think it was needed in both? @allenan ?

Reply:
I think it's fine as is, in dependencies